spec/mackerel-agent_spec.rb

```diff
@@ -5,7 +5,12 @@
 end
 
 describe service('mackerel-agent') do
-  it { should be_enabled   }
+  it {
+    if os[:family] == 'debian' && os[:release] >= '9.0'
+      pending('It seems serverspec cannot detect service status on docker debian 9...')
```

Comment:
I confirmed mackerel-agent service is enabled properly in Debian9 on VirtualBox.
```
vagrant@gemmyD9ChefTestvagrant01:/vagrant$ sudo systemctl status mackerel-agent
● mackerel-agent.service - mackerel.io agent
   Loaded: loaded (/lib/systemd/system/mackerel-agent.service; enabled; vendor preset: enabled)
   Active: inactive (dead)
     Docs: https://mackerel.io/
```